guides/advanced_guides/filtering.md

```diff
@@ -0,0 +1,131 @@
+# Filtering
+
+A search filter is a parameter that can be applied to the search query to refine the search results of a particular field (e.g. by size, color, price, or brand).
+Multiple filters may be applied to take a broad range of documents and refine them into a more narrow selection, allowing the end user to retrieve the most relevant search results based on the criteria they've selected.
```

Comment:
I'm not sure about what I said but this is to prepare the second guide and explain to users that we know that they are not the same and that we are working on the facetting.
```suggestion
Multiple filters(_expression_) may be applied to take a broad range of documents and refine them into a more narrow selection, allowing the end user to retrieve the most relevant search results based on the criteria they've selected.

Filters are processed at runtime, contrary to facetted search<Badge text="soon" type="warn"/>. Which gives filters great flexibility but will impact performance.
```

Reply:
It's maybe too early... Let me think of something.